src/Tickets/Site_Health/Subsections/Plugins/Plugin_Data_Subsection.php

```diff
@@ -0,0 +1,654 @@
+		$latest_rsvp_date = tribe( 'tickets.ticket-repository.rsvp' )
+			->per_page( 1 )
+			->order_by(
+				'date',
+				'DESC'
+			)
+			->pluck( 'post_date' );
+
+		// Check if we have any rsvp date.
+		if ( ! empty( $latest_rsvp_date ) ) {
+			$last_rsvp_creation_date = tribe_format_date(
+				$latest_rsvp_date[0],
+				true,
+				'Y-m-d'
+			);
+		} else {
+			$last_rsvp_creation_date = 'No rsvp found';
+		}
+		return $last_rsvp_creation_date;
+	}
+
+	/**
+	 * Gets the creation date of the latest attendee.
+	 *
+	 * @return string Date of the last attendee creation.
+	 */
+	private function get_last_attendee_creation_date(): string {
+		$latest_attendee_date = tribe( 'tickets.attendee-repository' )
+			->per_page( 1 )
+			->order_by(
+				'date',
+				'DESC'
+			)
+			->pluck( 'post_date' );
+
+		// Check if we have any attendee creation date.
+		if ( ! empty( $latest_attendee_date ) ) {
+			$last_attendee_creation_date = tribe_format_date(
+				$latest_attendee_date[0],
+				true,
+				'Y-m-d'
+			);
+		} else {
+			// @ Todo redscar need to possibly add translation.
+			$last_attendee_creation_date = "No Attendee's found";
+		}
+
+		return $last_attendee_creation_date;
+	}
+
+	/**
+	 * Retrieves the last app check-in date.
+	 *
+	 * @return string Last app check-in date in 'Y-m-d' format.
+	 */
+	private function get_last_app_check_in_date(): string {
+		return tribe_format_date(
+			tribe_get_option( 'tec_tickets_plus_app_last_checkin_time' ),
+			false,
+			'Y-m-d'
+		);
+	}
+
+	/**
+	 * Counts the total number of tickets.
+	 *
+	 * @return int Total number of tickets.
+	 */
+	private function get_number_of_tickets(): int {
+		return tribe( Tribe__Tickets__Query::class )->get_ticketed_count( 'tribe_events' );
+	}
+
+	/**
+	 * Counts the total number of RSVPs.
+	 *
+	 * @return int Total number of RSVPs.
+	 */
+	private function get_number_of_rsvps(): int {
+		return tribe( 'tickets.ticket-repository.rsvp' )->count();
+	}
+
+	/**
+	 * Counts the total number of attendees.
+	 *
+	 * @return int Total number of attendees.
+	 */
+	private function get_number_of_attendees(): int {
+		return tribe( 'tickets.attendee-repository' )->count();
+	}
+
+	/**
+	 * Computes and formats ticket prices, including average, max, and min prices.
+	 *
+	 * @return array Associative array with formatted average, max, and min ticket prices.
+	 */
+	private function get_formatted_prices(): array {
+		$ticket_prices = tribe( 'tickets.event-repository' )->per_page( -1 )->where( 'has_tickets' )->pluck( 'cost' );
+
+		$total_and_count = $this->calculate_total_and_count( $ticket_prices );
+		$max_price       = $this->calculate_max_price( $ticket_prices );
+		$min_price       = $this->calculate_min_price( $ticket_prices );
+		$average_price   = $this->calculate_average_price(
+			$total_and_count['total'],
+			$total_and_count['count']
+		);
+
+		return [
+			'formatted_max_price'     => number_format(
+				$max_price,
+				2,
+				'.',
+				','
+			),
```

Comment:
Everything looks really good! I just have a question about the number format. Should we be pulling the decimal point amount and separators (thousands and decimal) from their configuration?

Reply:
Good point. I updated it to use the currency util instead.